src/main/scala/gcewing/projectblue/mfr/MFRIntegration.java

```diff
@@ -0,0 +1,69 @@
+// ------------------------------------------------------------------------------------------------
+//
+// Project Blue - MineFactory Reloaded Integration
+//
+// ------------------------------------------------------------------------------------------------
+
+package gcewing.projectblue.mfr;
+
+import net.minecraft.block.*;
+import net.minecraft.init.*;
+import net.minecraft.item.*;
+
+import cpw.mods.fml.common.registry.*;
+import gcewing.projectblue.*;
+
+public class MFRIntegration extends gcewing.projectblue.BaseIntegration {
```

Comment:
Does anything actually initialize this class? I can't see references to it in this pr

Reply:
Seems to load the integration dynamically:

https://github.com/GTNewHorizons/ProjectBlue/blob/c8a5dfaf3197aa074e20b6255666d35d94baaf88/src/main/scala/gcewing/projectblue/ProjectBlue.java#L66-L71